package.json

```diff
@@ -2,6 +2,7 @@
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.10.2",
     "@rails/webpacker": "4.0.7",
+    "docker": "^1.0.0",
```

Comment:
Can you help me understand where these changes from? I'm a little surprised to see JS differences in here given the nature of this update. If this is doing something in particular, could we maybe roll it into a different PR?

For now, I'd like this change removed, and for there to not be changes to package-lock.json / yarn.lock, if that's cool!

Reply:
Sorry about that... not sure what happened... This is my first contribution to an open source project. I'll fix it and open a new PR.

Reply:
no apologies necessary! I do this all the time too,  it's why we have peer review on pull requests.